Helper/Identifier.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace NS8\Protect\Helper;
+
+use Magento\Customer\Model\Session as CustomerSession;
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\HTTP\Header;
+use Magento\Framework\HTTP\PhpEnvironment\Request;
+use Magento\Framework\Session\SessionManagerInterface;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Helper Utility for generating, managing, and validating Identifiers
+ */
+class Identifier extends AbstractHelper
+{
+    /**
+     * Generates a UUID
+     *
+     * @return string - A string containing a UUID
+     */
+    public function generateUUID(): string
+    {
+        if (function_exists('com_create_guid')) {
+            $merchantId = com_create_guid();
```

Comment:
I know very little about PHP (it's probably been 10+ years since I've done anything with it), but does `com_create_guid()` enclose the UUID in curly braces? I'm basing my question on [this page](https://www.php.net/manual/en/function.com-create-guid.php) in the PHP documentation, so let me know if this is using a different `com_create_guid()`.

Reply:
I think that doc is saying that the only difference between PHP's implementation and Microsoft's is that MS surrounds the UUID with `{}`, whereas PHP's impl does not

Reply:
@rob-ns8 @aldahick - I added a trim method surrounding this in the event that PHP invokes MS's generation logic 👍

Reply:
I think it would be safer to use an [existing library](https://packagist.org/packages/ramsey/uuid) for generating UUIDs rather than trying to reinvent the wheel here.

Reply:
I checked out Ramsey's UUID library but it also seems like a massive library with significant overhead for what we're trying to accomplish. In a sense, it seems as though we're adding a wheel with 36-inch gold rims and custom rubber stamps when we're really after a simply wheel for a Honda as shown [here](http://guid.us/GUID/PHP)

Reply:
@davehensleyns8 if you think we're ok with the additional module support and carrying the overhead for it, I'm totally open to doing so but want to make sure it's worth the commitment/support.

Reply:
I mean, there are [other options](https://packagist.org/packages/kodus/uuid-v4) as well that look to be much smaller (if less popular).

Reply:
@davehensleyns8 Fair point, I was hesitant to use less popular repos due to lack of support but the code is simplistic enough to manage. Thanks for pointing that out.